packages/react-native-bridge/android/react-native-bridge/src/main/res/layout/activity_gutenberg_web_view.xml

```diff
@@ -32,11 +32,45 @@
         android:layout_height="match_parent"
         android:visibility="visible"/>
 
-    <View
+    <LinearLayout
         android:id="@+id/foreground_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/white"/>
+        android:background="@color/white"
+        android:gravity="center"
+        android:textAlignment="center"
```

Comment:
np: I believe this can be removed.

Reply:
I tested and confirmed that removing the `textAlignment` didn't have any negative side effects or prevent the text from being centred, so have removed it in https://github.com/WordPress/gutenberg/pull/32395/commits/7ebf60e54a446ee55bafae1235b0fa9bd2518e3b. Thank you!